src/main/webapp/script.js

```diff
@@ -95,3 +95,14 @@ function createListElement(item) {
   liElement.innerHTML = '<a href=\"' + item.profilePageUrl + '\">' + item.username + '</a>';
   return liElement;
 }
+
+// Show fail message if appropriate.
```

Comment:
Shorten to "Show fail message"
The method itself does not contain the logic to check "if appropriate", so it's a bit misleading.

Reply:
Actually it does, if fail isn't in the query string, it will do nothing and keep the fail message hidden.

Reply:
OH I got confused bc of the URL check got it

Reply:
i am a clown thank u sir